src/com/facebook/buck/jvm/java/coverage/ReportGenerator.java

```diff
@@ -13,6 +13,7 @@
  *******************************************************************************/
 package com.facebook.buck.jvm.java.coverage;
 
+import org.codehaus.plexus.util.FileUtils;
```

Comment:
This utility provides most of the logic to include/exclude files via wildcard patterns. Reimplementing it will be a lot more work than using a standard utility that has been well tested already.